src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -1846,6 +1846,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return diagnostic Is Nothing OrElse diagnostic.Severity <> DiagnosticSeverity.Error
         End Function
 
+        ''' <summary>
+        ''' Returns true if there is a widening conversion from
+        ''' <paramref name="fromType"/> to <paramref name="toType"/>.
+        ''' </summary>
+        Public Overrides Function IsAssignableTo(fromType As ITypeSymbol, toType As ITypeSymbol) As Boolean
```

Comment:
>Public Overrides Function IsAssignableTo(fromType As ITypeSymbol, toType As ITypeSymbol) As Boolean [](start = 8, length = 99)

Same suggestion as for C#

Reply:
This region properly classifies the method.

---
In reply to: [186801342](https://github.com/dotnet/roslyn/pull/26719#discussion_r186801342) [](ancestors = 186801342)